content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -6,69 +6,86 @@ aliases:
 
-The Events Explorer replaces the Event Stream, one of Datadog's oldest features. The new Event query syntax requires migration of Event monitors, dashboard widgets, and any other features that query Events. 
+## Table of Contents
+- [Why the migration?](#why-the-migration)
+- [What is the migration timeline?](#what-is-the-migration-timeline)
+- [What action do I need to take?](#what-action-do-i-need-to-take)
+- [What's new?](#whats-new)
+- [What changed?](#what-changed)
+- [Examples](#examples)
 
-The introduction of the Events Explorer and its associated changes means that Event features work similarly to other Datadog products. Event query syntax, explorer and analytics views, processing pipelines and other features look like their counterparts in Logs or APM Traces. See [Events Explorer][1] for more details.
+## Why the migration?
 
-## What this means for your organization
+Launched more than 10 years ago, Datadog's event stream is one of our earliest features. Our new events experience includes many new features that allow you to get even more value from your events. These include event analytics, the ability to generate metrics from your events, and a much friendlier and more intuitive query syntax that is better aligned with other Datadog products, such as Log Management and APM.
 
-* New events (after migration) are visible only in the [Events Explorer][2] or dashboard widgets and event overlays.
+## What is the migration timeline?
 
-* Older events (before migration) continue to be visible in the [Event Stream][2] for the remainder of their retention period (13 months).
+<strong>March 5, 2022</strong> - The new Events Explorer and analytics were made available. We began to migrate customer dashboards and monitors not managed using our API.
 
-This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features. Following this guide ensures that you are ready when Datadog retires the legacy event stream and storage. This guide also provides [a detailed description of what's changed](#details-of-the-changes).
+<strong>April 30, 2022</strong> - The event stream will be retired in favor of the Event Explorer.
 
-## Migration process
+<strong>May 15, 2022</strong> - Starting on this date, while Datadog will continue to evaluate event monitors that have not been migrated, they will no longer be editable. New event monitors must use the new syntax.
 
-### Datadog migrates monitors and dashboards configured through the UI
-<div class="alert alert-info">
-You do not need to take any action for dashboards or monitors created through Datadog.
-</div>
+<strong>June 30, 2022</strong> - On this date, Datadog will stop evaluating event monitors that have not been migrated.
```

Comment:
Wouldn't be better to say "the old/legacy Events monitors" instead of "monitors not been migrated"? The result of the migration won't be a single "migrated" monitor, but an old monitor (muted) and a new version one evaluating.